src/DataAPI.jl

```diff
@@ -198,17 +198,24 @@ struct All{T<:Tuple}
 end
 
 """
-    Cols(cols...)
-    Cols(f::Function)
+    Cols(cols...; operator=union)
+    Cols(f::Function; operator=union)
 
-Select the union of the selections in `cols`. If `cols == ()`, select no columns.
+Select columns matching specifications in `cols`. If `cols == ()`, select no columns.
 
 If the only positional argument is a `Function` `f` then select the columns whose
 names passed to the `f` predicate as strings return `true`.
+
+When multiple `cols` selectors are passed then the sets of columns selected by them
+are passed to `operator` callable as positional arguments.
```

Comment:
```suggestion
are passed to `operator` as positional arguments.
```
(or "the `operator` callable", but the next sentence says it must be a function)